package.json

```diff
@@ -16,7 +16,7 @@
     "bcryptjs": "^2.4.3",
     "drizzle-kit": "^0.24.2",
     "drizzle-orm": "^0.33.0",
-    "next": "14.2.5",
+    "next": "^14.2.15",
```

Comment:
Don't change the version of next

Reply:
Hi @ajaynegi45, 
Regarding the design, I’ve strictly followed the Figma file provided, ensuring everything matched as closely as possible. If there were any enhancements needed, let me know. I also believe you had a chance to review the design from the video I shared, where I walked through the page.